c2rust-lib.rs

```diff
@@ -66,3 +57,93 @@ pub use bzlib::{
     BZ2_bzclose, BZ2_bzdopen, BZ2_bzerror, BZ2_bzflush, BZ2_bzlibVersion, BZ2_bzopen, BZ2_bzread,
     BZ2_bzwrite,
 };
+
+// --- version number logic
+
+macro_rules! libbzip2_rs_sys_version {
+    () => {
+        concat!("1.1.0-libbzip2-rs-sys-", env!("CARGO_PKG_VERSION"))
+    };
+}
+
+pub(crate) use libbzip2_rs_sys_version;
+
+// --- assert failure logic
+
+macro_rules! assert_h {
+    ($condition:expr, $errcode:expr) => {{
+        if !$condition {
+            eprint!("{}", $crate::AssertFail($errcode));
+            std::process::exit(3);
+        }
+    }};
+}
+
+pub(crate) use assert_h;
+
+pub(crate) struct AssertFail(i32);
+
+impl core::fmt::Display for AssertFail {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!(
+            concat!(
+                "\n",
+                "\n",
+                "bzip2/libbzip2: internal error number {}.\n",
+                "This is a bug in bzip2/libbzip2, {}.\n",
+                "Please report it at: https://gitlab.com/bzip2/bzip2/-/issues\n",
```

Comment:
This should probably be changed to point to this repo instead. And everywhere in the crash message references to bzip2 should be changed to bzip2-rs.